static/src/components/task_browse/assign_worker.vue

```diff
@@ -28,8 +28,7 @@
         </div>
       </div>
     </div>
-    <p>
-    </p>
+    <p/>
```

Comment:
Minor, is the empty paragraph element necessary in the template? If so, is the [self-closing tag](https://clearlydecoded.com/anatomy-of-html-tag) intended or should this be `<p></p>`?

Reply:
Originally I had <p></p>, and travis gave me a warning that Deep wanted me to fix. So this was what satisfied the warning